### PR TITLE
[Doc] add helm

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [x] 2.4
  - [ ] 2.3

### CODE REVIEW

**Review by evelynzhaojie — DISMISSED** [review]

File: docs/TOC.md [hunk 1]
```diff
@@ -87,7 +87,9 @@
   + Deployment
     + [Deploy a shared-data StarRocks cluster](./administration/deploy_shared_data.md)
     + [Deploy StarRocks with StarGo](./administration/stargo.md)
-    + [Deploy and manage CN on Kubernetes with StarRocks Operator](./administration/sr_operator.md)
+    + Deploy StarRocks on Kubernetes
+      + [Deploy StarRocks with operator](./administration/sr_operator.md)
```
Comment (amber-create) [inline docs/TOC.md:91]: image
应该大写为 Operator 吧

File: docs/administration/helm.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+[Helm](https://helm.sh/) is a package manager for Kubernetes. A [Helm Chart](https://helm.sh/docs/topics/charts/) is a Helm package and contains all of the resource definitions necessary to run an application on a Kubernetes cluster. This topic describes how to use Helm to automatically deploy a StarRocks cluster on a Kubernetes cluster.
+
+## Before you begin
+
+- [Create a Kubernetes cluster](../sr_operator#create-kubernetes-cluster).
+- [Install Helm](https://helm.sh/docs/intro/quickstart/).
+
+## Procedure
+
+> NOTICE
```
Comment (amber-create) [inline docs/administration/helm.md:12]: 加个粗

File: docs/administration/helm.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+   3. View the Helm Chart Repo that you added.
+
+      ```Bash
+      $ helm search repo starrocks-community
+      NAME                                      CHART VERSION        APP VERSION        DESCRIPTION                                       
+      starrocks-community/kube-starrocks        1.6.1                2.5.4              kube-starrocks collects Kubernetes manifests, s...
+      ```
+
+2. Use the default **[values.yaml](https://github.com/StarRocks/helm-charts/blob/main/charts/kube-starrocks/values.yaml)** of the Helm Chart to deploy the StarRocks Operator and StarRocks cluster, or create a YAML file to customize your deployment configurations.
+   1. Deploy with default configurations
```
Comment (amber-create) [inline docs/administration/helm.md:38]: 这句后面需要换个行，不然跟下一段 Run xxx 连到一起了。

File: docs/administration/helm.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+      Run the following command to deploy the StarRocks Operator and the StarRocks cluster that consists of one FE and one BE:
```
Comment (amber-create) [inline docs/administration/helm.md:39]: image
限定性定语从句和非限定性定语从句表达的含义不一样，比如这里，应该用非限定性定语从句，可改为 -> xxx the StarRocks cluster which consists of xxx.

File: docs/administration/helm.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    Deployment takes a while. During this period, you can check the deployment status by using the prompt command in the returned result of the deployment command above. The default prompt command is as follows:
+
+    ```Bash
+    $ kubectl --namespace default get starrockscluster -l "cluster=kube-starrocks"
+    # If the following result is returned, the deployment has been successfully completed.
+    NAME             FESTATUS   CNSTATUS   BESTATUS
+    kube-starrocks   running               running
+    ```
+
+    You can also run `kubectl get pods` to check the deployment status. If all Pods are in the`Running` state and all containers within the Pods are `READY`, the deployment has been successfully completed.
```
Comment (amber-create) [inline docs/administration/helm.md:75]: image
中间少了空格

File: docs/administration/helm.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    $ kubectl get pods
+    NAME                                       READY   STATUS    RESTARTS   AGE
+    kube-starrocks-be-0                        1/1     Running   0          2m50s
+    kube-starrocks-fe-0                        1/1     Running   0          4m31s
+    kube-starrocks-operator-69c5c64595-pc7fv   1/1     Running   0          4m50s
+    ```
+
+## Next steps
+
+**Access StarRocks cluster**
```
Comment (amber-create) [inline docs/administration/helm.md:87]: 下面少空行，会导致发布后跟下面一段连到一起

File: docs/administration/helm.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+> The StarRocks cluster is located in the namespace `default`.
+
+**Manage StarRocks operator and StarRocks cluster**
+
+- If you need to update the configurations of the StarRocks operator and StarRocks cluster, see [Helm Upgrade](https://helm.sh/docs/helm/helm_upgrade/).
+- If you need to uninstall the StarRocks Operator and StarRocks cluster, run the following command:
+
+  ```bash
+  helm uninstall starrocks
+  ```
```
Comment (kevincai) [inline docs/administration/helm.md:101]: can also add a reference link to our helm chart on artifact hub: https://artifacthub.io/packages/helm/kube-starrocks/kube-starrocks

**Review by amber-create — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-3.0  branch-2.5 branch-2.4

**Pull request merged**